src/spring/azext_spring/app.py

```diff
@@ -78,6 +87,17 @@ def app_create(cmd, client, resource_group, service, name,
     logger.warning(LOG_RUNNING_PROMPT)
     _ensure_app_not_exist(client, resource_group, service, name)
     sku = get_spring_sku(client, resource_group, service)
+
+    if sku.tier.upper() == 'STANDARDGEN2':
+        if cpu is None and memory is None:
+            cpu = '500m'
+            memory = '1Gi'
```

Comment:
Why isn't this logic like this?
```
       if cpu is None:
            cpu = '500m'
       if memory is None:
            memory = '1Gi'
```

Reply:
Good question, actually, the background is that there is a hard limit for new tier - StandardGen2 that the value of Memory must be double value of CPU, it means that, for example, if the cpu is 500m (0.5 cpu), the memory must be 0.5 * 2 = 1 Gi. 

The logic here is that if customer don't input any value for cpu and memory, we will use the default value (500m, 1Gi)，but if they input any value of CPU or Memory, we won't do some special logic to handle the other one, which will be handled by our backend service.

Reply:
Got it, thanks~